### PR TITLE
Integration spec

### PR DESCRIPTION
**Pull request by Paskab012**
Hello 👋  Reviewer, 
## In this PR I have 
I have added tests using Capybara to the views:
- Login page(where the username/password, and the `Submit button`)
- User index page.
- User show page.
- Post index page.
- Post show page.

### CODE REVIEW

File: spec/integration/login_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
```
Comment (hiromataba) [inline spec/integration/login_spec.rb:1]: - [x] When I am running your test they are not passing, Please add more instructions on how to run your tests or fix the issue 😉

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Paskab012     :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/integration/login_spec.rb (hunk 1 above)
Comment (descholar-ceo) [reply spec/integration/login_spec.rb:1]: 👍🏽 Addressed ✔️

**Review by descholar-ceo — APPROVED** [review]
Hi @Paskab012 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Addressed requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**